src/Time/Time.js

```diff
@@ -104,86 +116,96 @@ class Time extends Component {
                 id={id}>
                 {/* Hours */}
                 {showHour ? (
-                    <TimeItem
-                        defaultValue={1}
-                        disabled={disabled}
-                        disableStyles={disableStyles}
-                        downButtonProps={hoursDownButtonProps}
-                        format12Hours={format12Hours}
-                        inputProps={hoursInputProps}
-                        localizedText={localizedText}
-                        max={max}
-                        name='hour'
-                        placeholder={'hh'}
-                        spinners={spinners}
-                        time={time}
-                        type={'Hours'}
-                        upButtonProps={hoursUpButtonProps}
-                        updateTime={this.updateTime}
-                        value={time.hour} />
+                    <div className='fd-time__col' onClick={() => this.updateActiveColumn('hour')}>
+                        <label className='fd-time__slider-label'>Hours</label>
```

Comment:
Should these labels be localized? Could we just add to `localizedText` and pull them out here?

Reply:
Thanks for catching that, I noticed the `TimeItem` was also missing localized text so I've updated them all.